appeals/web/environment/config.js

```diff
@@ -109,10 +110,11 @@ export function loadConfig() {
 				readerGroupId: APPEALS_READERS_GROUP_ID
 			}
 		},
-		// flag name convention: featureFlag[ jira number ][ferature shoret description]
-		// set Feature Flag default val here [default: false] - will be overwritted by values cming from the .env file
+		// flag name convention: featureFlag[ jira number ][feature short description]
+		// set Feature Flag default val here [default: false] - will be overwritted by values coming from the .env file
 		featureFlags: {
-			featureFlagBoas1TestFeature: FEATURE_FLAG_BOAS_1_TEST_FEATURE === 'true'
+			featureFlagBoas1TestFeature: FEATURE_FLAG_BOAS_1_TEST_FEATURE === 'true',
```

Comment:
Should this be removed?

Reply:
No it is currently used for feature-flag unit test